src/disclosures/js/views/metric-view.js

```diff
@@ -107,86 +126,94 @@ var metricView = {
-   * @param {object} $nationalPoint jQuery object of the graph's national point
+   * @param {number|NaN} schoolValue Value reported by the school
+   * @param {number|NaN} nationalValue Average national value
+   * @param {object} $school jQuery object of the graph's school point
+   * @param {object} $national jQuery object of the graph's national point
    */
-  setGraphPositions: function( $graph, schoolAverage, nationalAverage,
-    $schoolPoint, $nationalPoint ) {
+  setGraphPositions: function( $graph ) {
+    // schoolValue, nationalValue, $school, $national
     var graphHeight = $graph.height(),
-        minValue = $graph.attr( 'data-graph-min' ),
-        maxValue = $graph.attr( 'data-graph-max' ),
-        bottoms = this.calculateBottoms( minValue, maxValue, graphHeight,
-          schoolAverage, nationalAverage );
+        metricKey = $graph.attr( 'data-metric' ),
+        nationalValue = metricView.metrics[metricKey].national,
+        schoolValue = metricView.metrics[metricKey].school,
+        min = $graph.attr( 'data-graph-min' ),
+        max = $graph.attr( 'data-graph-max' ),
+        $school = $graph.find( '.bar-graph_point__you' ),
+        $national = $graph.find( 'bar-graph_point__average' ),
+        bottoms = {},
+        bottomOffset = 20;
+
+    bottoms.school = ( graphHeight - bottomOffset ) / ( max - min ) *
+      ( schoolValue - min ) + bottomOffset;
+    bottoms.national = ( graphHeight - bottomOffset ) /
+      ( max - min ) * ( nationalValue - min ) + bottomOffset;
+
     // A few outlier schools have very high average salaries, so we need to
     // prevent those values from falling off the top of the graph
-    if ( schoolAverage > maxValue ) {
+    if ( schoolValue > max ) {
       bottoms.school = graphHeight;
       $graph.addClass( 'bar-graph__high-point' );
     }
-    $schoolPoint.css( 'bottom', bottoms.school );
-    $nationalPoint.css( 'bottom', bottoms.national );
+    $school.css( 'bottom', bottoms.school );
+    $national.css( 'bottom', bottoms.national );
   },
 
   /**
    * Classifies school value in relation to the national average
-   * @param {number|NaN} schoolValue Value reported by the school
-   * @param {number|NaN} nationalValue Average national value
-   * @param {number} sameMin Lowest value considered "about the same" as the
-   * national average (from College Scorecard)
-   * @param {number} sameMax Highest value considered "about the same" as the
-   * national average (from College Scorecard)
-   * @param {string} betterDirection 'higher' or 'lower' depending on whether a
-   * school value higher or lower than the national average is more desirable
+   * @param {number|NaN} metricKey - metric to be checked
    * @returns {string} Classes to add to the notification box
    */
-  getNotificationClasses: function( schoolValue, nationalValue, sameMin,
-    sameMax, betterDirection ) {
-    var notificationClasses = '';
+  getNotifications: function( metricKey ) {
```

Comment:
> gulp lint
  190:21  warning  Function 'getNotifications' has a complexity of 11

😲 

The code refactor is *increasing* the cyclomatic complexity of this section, and into the double digits. As much as I enjoy Spinal Tap, I'm pretty uncomfortable with the idea of going to 11. Could we use a different approach in the code to keep from doing that?